### ISSUE TITLE
Button container border radius behaviour change

### ISSUE THREAD

**Issue by chrisgreen1993** [react-native-elements/react-native-elements#3040]
First i'd like to say that I really appreciate this library, it's made my life much easier ❤️ 

**Explain what you did (Required)**

Create a `Button` with a `buttonStyle` with `borderRadius`. 

`containerStyle` added so you can easily see the different behaviour.  E.g:
```
<Button buttonStyle={{ borderRadius: 20 }} containerStyle={{ backgroundColor: 'red' }} />
```

**Expected behavior (Required)**

In v3.2.0 and below the `Button` container takes the `borderRadius` set in `buttonStyle`.

![Screenshot 2021-05-18 at 15 00 38](https://user-images.githubusercontent.com/1177794/118665014-dabcc780-b7e9-11eb-8d3b-30e6a28accf5.png)

**Describe the bug (Required)**

However since v3.3.0 this is no longer the case:

![Screenshot 2021-05-18 at 15 02 04](https://user-images.githubusercontent.com/1177794/118665242-1061b080-b7ea-11eb-9d7e-0c1a78135993.png)

**To Reproduce (Required)**

https://snack.expo.io/p7EUzV-h2

1. Go to snack above.
2. Note how the button container doesn't have a border radius and you can see the red background
3. Go to package.json and change version of react-native-elements to `3.2.0`.
4. The container now has a border radius.

**Your Environment (Required):**

| software              | version |
| --------------------- | ------- |
| react-native-elements |    3.4.1     |
| react-native          |     0.63.3    |

I'm unsure if this is actually a bug as I can fix the issue by also setting a borderRadius in the containerStyle, but it appears to have been introduced in this commit here: 

https://github.com/react-native-elements/react-native-elements/pull/2781/commits/5d4995f72d7b13cdd21b9a02dfaf709dd30ed5ce#diff-a4a4eaf5f85ed40c14a570378492560d1ca2695473ec3cc49ae4e2e90cad68b0R144

Let me know if it is a bug and I can submit a PR.

**Comment by arpitBhalla** [comment]
You can do this
```
  <View style={styles.container}>
     <Button
      title="Testing"
      containerStyle={{
        backgroundColor: 'red',
        borderRadius: 20,
      }}
/>
```

**Comment by arpitBhalla** [comment]
You have to use this 
```
     <Button
      title="Testing"
      containerStyle={{
        backgroundColor: 'red',
        borderRadius: 20,
      }}/>
```

**Comment by NaitikMishra** [comment]
Actually @chrisgreen1993 you have separated them by defining container style and again button style so they are implemented separately . I think in 3.2.0 it take don,t take separately . This might be considered as error as defining two property separately should perform separately and this is corrected in 3.3.0  .If you want to define separate then use 
```tsx
containerStyle={{
        backgroundColor: 'red',
        borderRadius: 20,
      }}
      buttonStyle={{
        borderRadius: 20,
      }}
``` 
other button can be implemented in this .
